nautobot/ipam/views.py

```diff
@@ -234,7 +234,7 @@ def get_extra_context(self, request, instance):
 
 
 class VRFListView(generic.ObjectListView):
-    queryset = VRF.objects.all()
+    queryset = VRF.objects.select_related("namespace", "tenant")
```

Comment:
Should we prefetch tags, import targets and export targets here? They're not default columns but they are available columns.